doc/user-guide/muc_light.md

```diff
@@ -0,0 +1,1821 @@
+# 1. Introduction
+
+Classic Multi-User chat, as described in XEP-0045, adds an IRC-like functionality to XMPP. It distinguishes between the affiliation list and the occupant list, where the latter is based on presences routed to the room from the client resource. While perfectly sufficient for desktop applications and relatively stable network connection, it does not exactly meet the challenges the mobile world is facing. Modern mobile applications do not rely on presence information, as it can frequently change. The expected user experience not only differs from IRC model, but also uses only a small subset of XEP-0045 features. The service described in this specification attempts to provide a complete solution for all common use cases of mobile groupchats.
+
+# 2. Requirements
+
+The list below contains the high-level features required from a new variant of MUC
+
+1. The service allows any user to create a room for group communication.
+1. Users cannot join rooms on their own. They have to be added by the room owner or (if configured by service administrator) any other occupant.
+1. Only the owner can remove other occupants from the room.
+1. Every occupant can leave the room.
+1. A user may block the attempts of being added to the specific room or by specific user.
+1. The message sent in the room is always broadcasted to every occupant.
+1. The full occupant list is always available to all occupants.
+1. Occupant is always visible on the list, even if it does not have any resources online.
+1. Occupants can only have two affiliations: owner and member.
+1. There must be at most one owner in the room (the service can choose to treat all users equally).
+1. If the room becomes empty, it is destroyed.
+1. Occupants cannot hide behind nicks. Their real bare JID is always visible to everyone
+1. No exchange of any <presence/> stanza inside the room.
+1. The user MUST be able to retrieve the list of rooms he or she occupies.
```

Comment:
Why do you think so? It's taken from the original protoXEP created by @fenek. Words like `MUST`, `SHOULD`, `MAY` are usually uppercased in documents describing a standard / protocol.